code/modules/jobs/job_types/blueshield.dm

```diff
@@ -0,0 +1,46 @@
+/datum/job/blueshield
+	title = "Blueshield"
+	flag = BLUESHIELD
+	auto_deadmin_role_flags = DEADMIN_POSITION_HEAD|DEADMIN_POSITION_SECURITY
+	department_head = list("Captain")
+	department_flag = ENGSEC
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the Captain"
+	selection_color = "#ddddff"
+	req_admin_notify = TRUE
+	space_law_notify = TRUE
+	exp_requirements = 300
+	exp_type = EXP_TYPE_CREW
+	exp_type_department = EXP_TYPE_SECURITY
+
+	outfit = /datum/outfit/job/blueshield
+
+	access = list(ACCESS_BLUESHIELD, ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_COURT, ACCESS_MEDICAL, ACCESS_ENGINE, ACCESS_MAINT_TUNNELS, ACCESS_EXTERNAL_AIRLOCKS, ACCESS_TELEPORTER, ACCESS_HEADS, ACCESS_CAPTAIN, ACCESS_ROBOTICS, ACCESS_CMO, ACCESS_RD, ACCESS_QM, ACCESS_WEAPONS, ACCESS_CE, ACCESS_HOP, ACCESS_HOS, ACCESS_KEYCARD_AUTH, ACCESS_CLONING)
+	minimal_access = list(ACCESS_BLUESHIELD, ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG	, ACCESS_COURT, ACCESS_MEDICAL, ACCESS_ENGINE, ACCESS_MAINT_TUNNELS, ACCESS_EXTERNAL_AIRLOCKS, ACCESS_TELEPORTER, ACCESS_HEADS, ACCESS_CAPTAIN, ACCESS_ROBOTICS, ACCESS_CMO, ACCESS_RD, ACCESS_QM, ACCESS_WEAPONS, ACCESS_CE, ACCESS_HOP, ACCESS_HOS, ACCESS_KEYCARD_AUTH, ACCESS_CLONING)
```

Comment:
wouldn't keycard auth allow a single head with a blueshield to red alert or do other stuff keycards are supposed to require two heads to agree on for

Reply:
yes I didnt mean to add that woopsies